react/src/hooks/utils.tsx

```diff
@@ -14,6 +14,19 @@ import { Query, SetDataOptions } from "react-query/types/core/query";
 import { stringToBoolean } from "../functions";
 import { QueryWithSearchOptions as MTQueryWithSearchOptions } from "../typings";
 
+function getActiveEndpoint(): string {
+    const activeEndpoint = localStorage.getItem("endpoint");
+    return activeEndpoint === null ? "" : activeEndpoint;
+}
+
+/**
+ * Wrapper for fetch - injects currently selected endpoint url to the outgoing request
+ */
+export function customFetch(url: string, ...args: any[]): Promise<Response> {
```

Comment:
Is there a more expressive name for this function? `apiFetch`? Maybe the real issue is our proliferation of `fetch`s-- `changeFetch`, `basicFetch`, `customFetch`. Is it possible to merge `basicFetch` and `customFetch`?

Reply:
Renamed `customFetch` to `apiFetch`